stable/pomerium/templates/proxy-service.yaml

```diff
@@ -11,9 +11,15 @@ metadata:
 {{- if .Values.service.labels }}
 {{ toYaml .Values.service.labels | indent 4 }}
 {{- end }}
-{{- if .Values.service.annotations }}
-  annotations:
-{{ toYaml .Values.service.annotations | indent 4 }}
+annotations:
+{{- if .Values.proxy.service.annotations }}
+ {{- range $key, $value := .Values.proxy.service.annotations }}
+   {{ $key }}: {{ $value | quote }}
+ {{- end }}
+{{- else if .Values.service.annotations }}
+ {{- range $key, $value := .Values.service.annotations }}
+   {{ $key }}: {{ $value | quote }}
+ {{- end }}
```

Comment:
@travisgroth why not use:
```
{{- if .Values.proxy.service.annotations }}
{{ toYaml .Values.proxy.service.annotations | indent 4 }}
{{- else if .Values.service.annotations }}
{{ toYaml .Values.service.annotations | indent 4 }}
{{- end }}
```

Reply:
Offhand I'm not totally sure if there's a trade-off there.  I see this pattern in other charts, so that was what I recommended in  https://github.com/pomerium/pomerium-helm/pull/38

Ref: https://github.com/helm/charts/blob/master/stable/nginx-ingress/templates/controller-service.yaml

Reply:
Interesting, thx, I suppose the advantage is that you are sure that the value are quoted (since annotations MUST be string).